backend/repopulate_db.py

```diff
@@ -2,25 +2,38 @@
 # trees for us. Feel free to add more trees here.
 
 from pymongo import MongoClient
-from db_interface import VERWIKI_DB_NAME, TREES_TABLE_NAME
+from db_interface import VERWIKI_DB_NAME, TREES_TABLE_NAME, LINKS_TABLE_NAME
 
 if __name__ == "__main__":
 
     client = MongoClient("localhost", port=27017, serverSelectionTimeoutMS=1000)
     # Select the radialTrees collection (Similar to a table in SQL)
     radialTrees = client[VERWIKI_DB_NAME][TREES_TABLE_NAME]
+
+    # select the nodeLinks collection
+    nodeLinks = client[VERWIKI_DB_NAME][LINKS_TABLE_NAME]
+
     # Clear out all the old data
     deleted_data = radialTrees.delete_many({})
+    deleted_links = nodeLinks.delete_many({})
 
     # Tree for testing purposes
     tree1 = {
         "id": 1,
         "data": {"name": "Root", "children": [{"name": "Front", "children": []}]},
     }
 
+    # Links for testing purposes
```

Comment:
Why are we merging links for testing purposes into the main codebase?

Reply:
So these links (and this entire file in general) is solely used as a convenient way to repopulate our local databases to make it easier to develop locally; so this link is the one we read from in order to populate the side info text box when the user right clicks on a node. This file and this data will not be used at all in the actual release, it is just for us. I have updated the comments for this file to try to be more clear